src/hw/signMessage/index.js

```diff
@@ -90,8 +95,9 @@ export const createAction = (
         return;
       }
       try {
-        result = await withDevice(device.deviceId)((t) =>
-          from(dispatch(t, request.message))
+        result = await (
+          signMessage(request) ||
+          withDevice(device.deviceId)((t) => from(dispatch(t, request.message)))
```

Comment:
according to the type, the signMessage can't be "falsy", i understand this is made for deprecation then.
can you add an explicit comment here to say "DEPRECATED: update client of live-common to explicit the signMessage implementation"

Reply:
actually i did not think that much. I did something that should be better now